sdk/core/azure-core/inc/azure/core/http/http.hpp

```diff
@@ -91,22 +91,77 @@ namespace Azure { namespace Core { namespace Http {
   };
 
   /**
-   * HTTP request method.
-   *
+   * @brief The method to be performed on the resource identified by the Request.
```

Comment:
I like the description here: no "defines" or "provides", we just go straight and say what the class **is**.

Reply:
"...identified by HTTP request"?

Reply:
```suggestion
   * @brief The method to be performed on the resource identified by the HTTP request.
```

Reply:
Request is linked to the class:
![image](https://user-images.githubusercontent.com/24213737/119565183-80c67f80-bd5e-11eb-9b13-f20a48ea65c1.png)